test/feature/test_laf.py

```diff
@@ -250,3 +250,30 @@ def test_gradcheck(self):
         assert gradcheck(kornia.feature.extract_patches_from_pyramid,
                          (img, nlaf, PS,),
                          raise_exception=True)
+
+
+class TestConvertFromOpenCVKpts:
+    def test_shape(self):
+        class KP():
+            def __init__(self, x, y, scale, angle):
```

Comment:
No duplication code. Define this class outside or in a separated utils file.

Reply:
why not directly in the module with _?